hooks/backup/50-conf_manually_modified_files

```diff
@@ -18,7 +18,13 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 
+# Exit hook on subcommand error or unset variable
+set -Eeuo pipefail
+
+# Source YNH helpers
+# shellcheck source=../../helpers/helpers
```

Comment:
o_O

Reply:
I don't understand if it's really needed to add `shellcheck source=../../helpers/helpers`

Reply:
The linter doesn't understand what `source /usr/share/yunohost/helpers` means: we usually don't develop on our yunohost machines.
We need to tell shellcheck where to find the helpers, and that is a relative path to the current file.